model/simulationtests/dist_ht_steam_cl.rb

```diff
@@ -0,0 +1,242 @@
+chilled_water_temp_sch.winterDesignDaySchedule.addValue(os_time, 6.7)
+# Summer Design Day
+chilled_water_temp_schSummer = OpenStudio::Model::ScheduleDay.new(model)
+chilled_water_temp_sch.setSummerDesignDaySchedule(chilled_water_temp_schSummer)
+chilled_water_temp_sch.summerDesignDaySchedule.setName('Chilled_Water_Temperature_Summer_Design_Day')
+chilled_water_temp_sch.summerDesignDaySchedule.addValue(os_time, 6.7)
+# All other days
+chilled_water_temp_sch.defaultDaySchedule.setName('Chilled_Water_Temperature_Default')
+chilled_water_temp_sch.defaultDaySchedule.addValue(os_time, 6.7)
+
+# new airloop
+airLoopHVAC = OpenStudio::Model::AirLoopHVAC.new(model)
+
+# system sizing
+sizingSystem = airLoopHVAC.sizingSystem
+sizingSystem.setCentralCoolingDesignSupplyAirTemperature(12.8)
+sizingSystem.setCentralHeatingDesignSupplyAirTemperature(12.8)
+
+# fan
+fan = OpenStudio::Model::FanVariableVolume.new(model, always_on_sch)
+fan.setPressureRise(500)
+
+# hot water heating coil
+coilHeatingWater = OpenStudio::Model::CoilHeatingWater.new(model, always_on_sch)
+
+# chilled water cooling coil
+coilCoolingWater = OpenStudio::Model::CoilCoolingWater.new(model, always_on_sch)
+
+# setpoint managers
+setpointMMA1 = OpenStudio::Model::SetpointManagerMixedAir.new(model)
+setpointMMA2 = OpenStudio::Model::SetpointManagerMixedAir.new(model)
+setpointMMA3 = OpenStudio::Model::SetpointManagerMixedAir.new(model)
+deckTempSPM = OpenStudio::Model::SetpointManagerScheduled.new(model, deck_temp_sch)
+
+# OA controller
+controllerOutdoorAir = OpenStudio::Model::ControllerOutdoorAir.new(model)
+outdoorAirSystem = OpenStudio::Model::AirLoopHVACOutdoorAirSystem.new(model, controllerOutdoorAir)
+
+# add the equipment to the airloop
+supplyOutletNode = airLoopHVAC.supplyOutletNode
+outdoorAirSystem.addToNode(supplyOutletNode)
+coilCoolingWater.addToNode(supplyOutletNode)
+coilHeatingWater.addToNode(supplyOutletNode)
+fan.addToNode(supplyOutletNode)
+node1 = fan.outletModelObject.get.to_Node.get
+deckTempSPM.addToNode(node1)
+node2 = coilHeatingWater.airOutletModelObject.get.to_Node.get
+setpointMMA1.addToNode(node2)
+node3 = coilCoolingWater.airOutletModelObject.get.to_Node.get
+setpointMMA2.addToNode(node3)
+node4 = outdoorAirSystem.mixedAirModelObject.get.to_Node.get
+setpointMMA3.addToNode(node4)
+
+# Hot Water Plant
+hotWaterPlant = OpenStudio::Model::PlantLoop.new(model)
+sizingPlant = hotWaterPlant.sizingPlant
+sizingPlant.setLoopType('Heating')
+sizingPlant.setDesignLoopExitTemperature(82.0)
+sizingPlant.setLoopDesignTemperatureDifference(11.0)
+hotWaterOutletNode = hotWaterPlant.supplyOutletNode
+hotWaterInletNode = hotWaterPlant.supplyInletNode
+hotWaterDemandOutletNode = hotWaterPlant.demandOutletNode
+hotWaterDemandInletNode = hotWaterPlant.demandInletNode
+
+# pump
+pump = OpenStudio::Model::PumpVariableSpeed.new(model)
+
+# district heating steam
+district_heating_steam = OpenStudio::Model::DistrictHeatingSteam.new(model)
```

Comment:
This isn't testing any fields of the objects.

There's a lot of boilerplate for the rest.

Reply:
Also, if you want a PlantLoop with a DistrictHeating:Steam, you need the fluid type to Steam.
We'd need a Coil:Heating:Steam coil as well I think, except there aren't any in the SDK yet.

Reference ExampleFile: `SteamSystemAutoSize_DistrictHeatingSteam.idf`

(We probably could have just not wrapped this object TBH... it's not useful without the rest).

Maybe it's possible to hook it up to a HX that does Steam Loop => HW loop? Or jsut use a LoadProfile:Plant on the demand side.

I'm not going to waste energy on this _right now_ though

Reply:
I tried the LoadProfile:Plant.

We need a Pump:VariableSpeed:Condensate, and some Pipe:Adiabatic:Steam